### DISCUSSION TITLE
Navigate new Explore page for web version

### DISCUSSION THREAD

**Discussion by shine-jayakumar** [shine-jayakumar/insta-likecom-bot#98] [General]
Instagram allowed user to visit `https://www.instagram.com/explore/tags/tagname` (ex: `https://www.instagram.com/explore/tags/denver` and nagivate all the posts in the hastag.
Now it only shows few top n posts.
Few users are able to navigate to` https://www.instagram.com/explore/search/keyword/?q=<keyword here or tag> ` (ex:  https://www.instagram.com/explore/search/keyword/?q=denver) and view all the posts like before.
Welcoming suggestions for an alternative url or naviation path to view all posts in a hashtag.

**Comment by ap0l0** [comment]
Hi @shine-jayakumar 

Does the following steps give you any result?

1. Open https://www.instagram.com/explore/search/
<img width="1284" alt="image" src="https://github.com/shine-jayakumar/insta-likecom-bot/assets/14894967/bb8a101b-e7e8-47ac-917a-4839a07d2961">

2. Look for the tags on top
<img width="968" alt="image" src="https://github.com/shine-jayakumar/insta-likecom-bot/assets/14894967/efdf4e3c-e93b-4fdf-93d6-36bbbd563f1c">

3. Click on the second one in my case "python programming" (I am a horrible developer but IG thinks I can be better haha)
https://www.instagram.com/explore/search/keyword/?q=python%20programming

https://github.com/shine-jayakumar/insta-likecom-bot/assets/14894967/1bd31910-822a-4957-8c9c-06954664bde0

**Comment by ap0l0** [comment]
I have tried several direct URL's and I always get results. The logic is no longer a "tag" or word, but content search. For example:

https://www.instagram.com/explore/search/keyword/?q=openai
https://www.instagram.com/explore/search/keyword/?q=elon%20musk
https://www.instagram.com/explore/search/keyword/?q=surfing

<img width="1276" alt="image" src="https://github.com/shine-jayakumar/insta-likecom-bot/assets/14894967/85ff5908-480b-4994-bc47-5f97c60aff18">

I also added an alternative method of getting posts by location here #99

**Reply by shine-jayakumar** [reply]
This isn't working for me at all, even with different accounts and a VPN in place. I can't see the tags either on the top of the page like you get. It does have infinite scroll with lazy loading but random posts chosen with Instagram algorithm.

![Screenshot from 2023-12-03 12-16-07](https://github.com/shine-jayakumar/insta-likecom-bot/assets/84611104/ae8c3821-7bd1-4831-88f2-cc37158442ca)

![Screenshot from 2023-12-03 12-18-05](https://github.com/shine-jayakumar/insta-likecom-bot/assets/84611104/371770af-f2a4-4d64-9b94-c23ea0f69d3b)
![Screenshot from 2023-12-03 12-18-01](https://github.com/shine-jayakumar/insta-likecom-bot/assets/84611104/6805c0d0-b33c-4f36-9413-21e681a81f5b)

**Comment by digitizesuccess** [comment]
so what is happening when I run the following script(s) is basically the same thing

When I run the following script

"/usr/local/bin/python3 /Users/xxx/insta-likecom-bot/ilcbot.py -u xxx -p xxx -t denver -np 50 -lc 0 -nc -il 30d -lm /Users/xxxx/insta-likecom-bot/limits.json --delay 2,20 --ignoretags /Users/xxx/insta-likecom-bot/Ignorelist.txt"

It finds a page called @denver and it's just one page it doesn't bring up that search page. If I add quotes around Denver it brings up the same page. Other searches either bring up individual pages or you get an error for it not finding anything. If you add a # we run into the same problem that got us here in the first place.

However, what I noticed was this. You have to search the term and then click on a search result to bring up the main page. Please see the attached photo.
![Image 12-20-23 at 2 15 PM](https://github.com/shine-jayakumar/insta-likecom-bot/assets/67355464/057df932-3f23-4b7b-ad14-2e633c5832cd)

**Comment by digitizesuccess** [comment]
any updates?
